reference/5.0/PowershellGet/Find-Module.md

```diff
@@ -15,66 +15,83 @@ Finds modules from an online gallery that match specified criteria.
 
 ## SYNTAX
 
+### __AllParameterSets
+
 ```
-Find-Module [[-Name] <String[]>] [-MinimumVersion <Version>] [-MaximumVersion <Version>]
- [-RequiredVersion <Version>] [-AllVersions] [-IncludeDependencies] [-Filter <String>] [-Tag <String[]>]
- [-Includes <String[]>] [-DscResource <String[]>] [-Command <String[]>] [-Repository <String[]>]
- [<CommonParameters>]
+Find-Module [[-Name] <string[]>] [-MinimumVersion <version>] [-MaximumVersion <version>]
+[-RequiredVersion <version>] [-AllVersions] [-IncludeDependencies] [-Filter <string>]
+[-Tag <string[]>] [-Includes <string[]>] [-DscResource <string[]>] [-Command <string[]>]
+[-Repository <string[]>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Find-Module** cmdlet finds modules from the online gallery that match the specified criteria.
-**Find-Module** returns a **PSGetItemInfo** object for each module it finds, which you can be pipe to Install-Module to install.
 
-If the *RequiredVersion* parameter is not specified, **Find-Module** returns the latest version of the module that is equal to or greater than the minimum version specified, or the newest version of the module if no minimum version is specified.
+The `Find-Module` cmdlet finds modules from the online gallery that match the specified criteria.
+`Find-Module` returns a **PSRepositoryItemInfo** object for each module it finds, which you can be
+pipe to Install-Module to install.
+
+If the **RequiredVersion** parameter is not specified, `Find-Module` returns the latest version of
+the module that is equal to or greater than the minimum version specified, or the newest version of
+the module if no minimum version is specified.
 
-If the *RequiredVersion* parameter is specified, **Find-Module** only returns the version of the module that exactly matches the specified version.
-**Find-Module** searches through all available modules, because name conflicts between sources can occur.
+If the **RequiredVersion** parameter is specified, `Find-Module` only returns the version of the
+module that exactly matches the specified version. `Find-Module` searches through all available
+modules, because name conflicts between sources can occur.
 
 ## EXAMPLES
 
 ### Example 1: Find a module by name
+
 ```
 PS C:\> Find-Module -Name "ContosoServer"
```

Comment:
Remove the PS prompts from the examples throughout the article

Reply:
Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.